src/events.ts

```diff
@@ -0,0 +1,30 @@
+import { EventHandler } from "playcanvas";
```

Comment:
Any reason not to just import Events from `@playcanvas/observer`?

Reply:
Seems a pretty arbitrary choice. Is there a reason to prefer observer events?

Reply:
Well, mainly because I'm anticipating SuperSplat adopting our Observer, which subclasses the Observer Events class.

Reply:
Yeah makes sense, I'll make that change in a followup PR.